test/generator.cpp

```diff
@@ -207,22 +281,45 @@ int main(const int argc, const char** argv) {
 
     }
 
-    if (kind < 0 || d < 0) {
+    if (d < 1) {
         std::cout<<"Wrong inputs, try -help"<<std::endl;
         exit(-1);
     }
+    if (Hpoly && rand) kind = 6;
 
     if (Zono) {
         if (m > 0) {
-            Zonotope ZP = gen_zonotope<Zonotope, RNGType>(d, m);
-            create_txt(ZP.get_mat(), ZP.get_vec(), kind, true);
+            Zonotope ZP;
+            switch (kind) {
+                case -1:
```

Comment:
is it possible to get -1 ? why not putting a default case?

Reply:
I set a default choice.